src/components/ConfirmDeleteDialog..tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC, useState } from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogActions from '@material-ui/core/DialogActions';
+import Button from '@material-ui/core/Button';
+import ErrorBanner from './ErrorBanner';
+import { useTranslation } from 'react-i18next';
+import { deleteResource } from '../api/resourceApi';
+
+interface ConfirmDeleteDialogProps {
+  resourceIdentifier: string;
+  open: boolean;
+  confirmedDelete: () => void;
+  abortDelete: () => void;
+}
+
+const ConfirmDeleteDialog: FC<ConfirmDeleteDialogProps> = ({
+  resourceIdentifier,
+  open,
+  abortDelete,
+  confirmedDelete,
+}) => {
+  const { t } = useTranslation();
+  const [deleteErrorOccured, seDeleteErrorOccured] = useState(false);
+
+  const performDeletion = async () => {
+    try {
+      await deleteResource(resourceIdentifier);
+      seDeleteErrorOccured(false);
+      confirmedDelete();
+    } catch (error) {
+      seDeleteErrorOccured(true);
+    }
+  };
+
+  const performAbort = () => {
+    abortDelete();
+    seDeleteErrorOccured(false);
+  };
+
+  return (
+    <Dialog
+      open={open}
+      onClose={performAbort}
+      aria-labelledby="alert-dialog-title"
+      data-testid={`delete-confirm-dialog`}>
+      <DialogTitle id="alert-dialog-title">{`${t('resource.delete_resource_confirmation')}?`}</DialogTitle>
+      <DialogContent>
+        {!deleteErrorOccured && (
+          <DialogContentText id="alert-dialog-description">
+            {t('resource.delete_resource_description')}
+          </DialogContentText>
+        )}
+        {deleteErrorOccured && <ErrorBanner />}
+      </DialogContent>
+      <DialogActions>
+        <Button
+          data-testid={`delete-confirm-dialog-abort-button-${resourceIdentifier}`}
+          onClick={performAbort}
+          color="primary">
+          {t('common.cancel')}
+        </Button>
+        {!deleteErrorOccured && (
+          <Button
```

Comment:
Uppercase slik at knappene blir i samme stil som ellers.  Og kanskje fylt farge på "slett"-knappen slik at man lettere ser forskjell på de to. Color warning eller primary ?

Reply:
Fått endret til warning.
Figmaen så langt er textbutton: 
![image (3)](https://user-images.githubusercontent.com/44363838/105337034-3c717a00-5bda-11eb-957b-3d2227dfea33.png)

Reply:
jeg mente knappene i dialogen

Reply:
Hvordan stiler vi avbryt-knappen? UU-messig bør det være den man har lyst til å trykke på hvis man ikke følger med på hva man gjør men bare klikker rundt.

Reply:
jeg synes nå bekreft-knappen skal være mest synlig - det er jo den man i de fleste tilfeller skal trykke på.

Reply:
Fikset